tests/printf/test.out

```diff
@@ -23,11 +23,11 @@ INF
 nan
 NAN
        nan
-nan       
```

Comment:
Probably best to revert all files ending in ".out" in the tests directory.   These are expected output files and may contains significant whitespace as in this example.